tests/onnx_chainer_tests/functions_tests/test_arrays.py

```diff
@@ -155,6 +155,11 @@
      'args': {'slices': (slice(None), slice(0, 1), slice(None, 2))},
      'name': 'get_item_start_from_none'},
 
+    # select_item
```

Comment:
Could you split this test from `TestArrayOperators`, because this function is for unary args.

```
class TestSelectItem(ONNXModelTest):

    def test_output(self):

        class Model(chainer.Chain):

            def forward(self, x, t):
                return F.select_item(x, t)

        model = Model()
        x = input_generator.increasing(3, 3)
        t = np.array([2, 1, 0], dtype=np.int32)

        self.expect(
            model, (x, t), expected_num_initializers=0,
            skip_opset_version=(7, 8))
```

- set `t` as input, not to put unnecessary initializer
- set skip opset version